src/main/java/seedu/address/logic/commands/DeleteManyCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.logic.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Deletes a person identified using it's last displayed index from the address book.
+ */
+public class DeleteManyCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "deletem";
+    public static final String COMMAND_ALIAS = "dm";
+
```

Comment:
Should we extend the existing delete command to include this functionality instead of having a new command? I personally feel that this might be make it more user friendly as we would use the same command syntax as `delete`. 

Looking for inputs from rest of the team @CS2103AUG2017-W10-B2/developers